src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/RazorConfigurationJsonConverter.cs

```diff
@@ -24,10 +25,9 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 return null;
             }
 
-            var obj = JObject.Load(reader);
-            var configurationName = obj[nameof(RazorConfiguration.ConfigurationName)].Value<string>();
-            var languageVersion = obj[nameof(RazorConfiguration.LanguageVersion)].Value<string>();
-            var extensions = obj[nameof(RazorConfiguration.Extensions)].ToObject<RazorExtension[]>(serializer);
+            var configurationName = reader.ReadNextStringProperty(nameof(RazorConfiguration.ConfigurationName));
+            var languageVersion = reader.ReadNextStringProperty(nameof(RazorConfiguration.LanguageVersion));
+            var extensions = reader.ReadPropertyArray<RazorExtension>(serializer, nameof(RazorConfiguration.Extensions)).ToArray();
```

Comment:
Does casting this to an array make a significant difference in the profile? (vs. ToArraying it?)

Reply:
Its a marginal difference, but `ToArray()` is unnecessary here, so removed it. Thanks!